lib/cog_api/http/client.ex

```diff
@@ -134,20 +134,36 @@ defmodule CogApi.HTTP.Client do
     RelayGroups.delete(relay_group_id, endpoint)
   end
 
-  def relay_group_add_relay(relay_group_id, relay_id, %Endpoint{}=endpoint) do
-    RelayGroups.update_memberships(:add, relay_group_id, relay_id, endpoint)
+  def relay_group_add_relays_by_name(relay_group_name, relay_names, %Endpoint{}=endpoint) do
```

Comment:
@christophermaier what do you think about this pattern. It seems a lot clearer to me.